src/test/java/org/fundacion/jala/converter/core/ChecksumServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package org.fundacion.jala.converter.core;
+
+import org.fundacion.jala.converter.core.exceptions.ChecksumException;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import java.io.File;
+import java.io.IOException;
+import static org.fundacion.jala.converter.core.ChecksumService.getFileChecksum;
+import static org.fundacion.jala.converter.core.ChecksumService.repeatedChecksum;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+public class ChecksumServiceTest {
+
+    @Rule
+    public TemporaryFolder folder = new TemporaryFolder();
+    
+    @Test
+    public void getFileChecksumShouldReturnFileChecksum() throws IOException, ChecksumException {
+        File tempFile = folder.newFile("myFile.txt");
+        String actual = getFileChecksum(tempFile.getPath());
+        String expected = "d41d8cd98f00b204e9800998ecf8427e";
+        assertEquals(expected, actual);
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void getFileChecksumShouldThrowChecksumExceptionWithNullInput() throws ChecksumException {
```

Comment:
Test convention wasn't supposed to be.. all test name start with should? shouldGetChecksum....?